### DISCUSSION TITLE
How to decode JWT token in Golang?

### DISCUSSION THREAD

**Discussion by mpiorowski** [nextauthjs/next-auth#6291] [Help]
I know it's an JWE implementation, but all the solution i found are not working . Would be rly grateful for help.

**Comment by kaizerpwn** [comment]
Hi, I have the same problem. did you find any solution for this?

**Comment by poshak-rialtes** [comment]
I am stuck with the same thing with python backend.Did you guys try reverse enfineering the next-auth library code.

**Comment by kiwamizamurai** [comment]
I've created a example middleware with golang
https://github.com/nextauthjs/next-auth/discussions/12622
